source/NanostackRfPhyAT86RF215.cpp

```diff
@@ -299,7 +301,7 @@ static int8_t rf_extension(phy_extension_type_e extension_type, uint8_t *data_pt
             rf_conf_set_cca_threshold(*data_ptr);
             break;
         case PHY_EXTENSION_SET_802_15_4_MODE:
-            mac_mode = (phy_802_15_4_mode_t) *data_ptr;
+            mac_mode = (phy_802_15_4_mode_t) * data_ptr;
```

Comment:
This is read behind data_ptr. *data_ptr was correct, right?

Reply:
hmm, let me run style again